README.md

```diff
@@ -1,103 +1,74 @@
-# listenbrainz-server
+listenbrainz-server
+===================
 
-Server for the ListenBrainz project.
+Server for the ListenBrainz project
 
-[Website](https://listenbrainz.org) | [Beta website](https://beta.listenbrainz.org) | [Bug tracker](https://tickets.metabrainz.org/projects/LB/issues)
+[Website](https://listenbrainz.org) |
+[Documentation](https://listenbrainz.readthedocs.io) |
+[Beta website](https://beta.listenbrainz.org) |
+[Bug tracker](https://tickets.metabrainz.org/projects/LB/issues)
 
-## Installation
 
-*These instructions are meant to get you started quickly with development
-process. Installation process in production environment might be different.*
+## About
 
-### Prerequisites
+The ListenBrainz project is similar to the original AudioScrobbler®. Unlike the
+original project, ListenBrainz is open source and publishes its data as open
+data.
 
-In order to install ListenBrainz onto your machine or a VM, you will
-need to install:
+A team of former Last.fm and current MusicBrainz hackers created the first
+version of ListenBrainz in a weekend. Since the original project was created,
+technology has advanced at an incredibly rapid pace, which made re-creating the
+original project fairly straightforward. 
 
-* Docker engine: https://docs.docker.com/engine/installation/
-* Docker compose: https://docs.docker.com/compose/install/
+The project has two main goals:
 
+1. Allow users to preserve their existing Last.fm® data
+2. Make this incredibly useful music usage data available to the world
 
-### Configuration file
+For more information about this project and its goals, look at our
+[website](https://listenbrainz.org/), specifically the
+[goals page](https://listenbrainz.org/roadmap).
 
-Copy the file `listenbrainz/config.py.sample` to `listenbrainz/config.py`:
 
-    $ cp listenbrainz/config.py.sample listenbrainz/config.py
+## Development environment
 
-Next, register for a MusicBrainz application:
+These instructions help you get started with the development process.
+Installation in a production environment may be different.
 
-   `https://musicbrainz.org/account/applications`
+**Read the [development environment
+documentation](https://listenbrainz.readthedocs.io/en/latest/dev/devel-env.html 
+"Setting up a development environment - ListenBrainz documentation")**
 
-During registration set the callback url to
 
-   `http://<your_host>/login/musicbrainz/post`
+## Calculating statistics
 
-Where <your_host> is the DNS name or IP address of the machine running ListenBrainz.
-An example callback url if you're running the server on your own local machine would
-be
+ListenBrainz uses [Google BigQuery](https://cloud.google.com/bigquery/ "BigQuery
```

Comment:
This link doesn't render well in GH markdown? 
![screenshot_2017-11-12_14-21-59](https://user-images.githubusercontent.com/1837631/32697371-ef235794-c7b4-11e7-9277-7c10f26e8628.png)